app/js/disease-associations-table-directive.js

```diff
@@ -416,6 +422,7 @@ angular.module('cttvDirectives')
                                     "association_score.datatypes.affected_pathway",
                                     "association_score.datatypes.rna_expression",
                                     "association_score.datatypes.literature",
+                                    "association_score.datatypes.literature_nferx",
```

Comment:
i thought we had agreed they should be folded in the literature datatype.

Reply:
Yes, and in any case this shouldn't go in the public version